code/datums/status_effects/neutral.dm

```diff
@@ -270,3 +270,19 @@
 /datum/status_effect/delayed/on_timeout()
 	. = ..()
 	expire_proc.Invoke()
+
+/datum/status_effect/action_status_effect
+	alert_type = null
+	tick_interval = -1
+
+/datum/status_effect/action_status_effect/remove_handcuffs
+	id= "remove handcuffs"
+
+/datum/status_effect/action_status_effect/break_handcuffs
+	id= "break handcuffs"
+
+/datum/status_effect/action_status_effect/remove_muzzle
+	id= "remove muzzle"
+
+/datum/status_effect/action_status_effect/unbuckle
+	id= "unbuckle"
```

Comment:
instead of creating a bunch of empty status effects, couldnt you register a signal to listen for when someone gets examined when they start removing cuffs, muzzle, etc. and when it hears the signal show a message to the examiner?